cmd/aspect/query/cquery/cquery.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright © 2021 Aspect Build Systems
+
+Not licensed for re-use
+*/
+
+package cquery
+
+import (
+	"aspect.build/cli/pkg/aspect/query/cquery"
+	"aspect.build/cli/pkg/bazel"
+	"aspect.build/cli/pkg/ioutils"
+	"github.com/spf13/cobra"
```

Comment:
```suggestion
	"github.com/spf13/cobra"

	"aspect.build/cli/pkg/aspect/query/cquery"
	"aspect.build/cli/pkg/bazel"
	"aspect.build/cli/pkg/ioutils"
```

Reply:
Done. I still think we should have a linter that would catch these types of things

Reply:
You could try to use `goimports -local aspect.build/cli` to see if it does the right thing. If it does, feel free to add to https://github.com/aspect-build/aspect-cli/blob/main/.github/workflows/tidy_up_repository.sh.